app/models/setting/general.rb

```diff
@@ -19,7 +19,7 @@ def self.default_settings
       set('db_pending_migration', N_("Should the `foreman-rake db:migrate` be executed on the next run of the installer modules?"), true, N_('DB pending migration')),
       set('db_pending_seed', N_("Should the `foreman-rake db:seed` be executed on the next run of the installer modules?"), true, N_('DB pending seed')),
       set('proxy_request_timeout', N_("Open and read timeout for HTTP requests from Foreman to Smart Proxy (in seconds)"), 60, N_('Smart Proxy request timeout')),
-      set('login_text', N_("Text to be shown in the login-page footer"), nil, N_('Login page footer text')),
+      set('login_text', N_("Text to be shown in the login-page footer"), nil, N_('Login page footer text'), nil, {:field => 'textarea'}),
```

Comment:
Instead of option key `field`, why you are not passing `settings_type` key?

Reply:
Make sense in naming option key to `settings_type`. I will make the changes.